Mathlib/Data/ZMod/Units.lean

```diff
@@ -30,4 +30,10 @@ lemma unitsMap_comp {d : ℕ} (hm : n ∣ m) (hd : m ∣ d) :
 lemma unitsMap_self (n : ℕ) : unitsMap (dvd_refl n) = MonoidHom.id _ := by
   simp [unitsMap, castHom_self]
 
+lemma IsUnit_cast_of_dvd (hm : n ∣ m) (a : Units (ZMod m)) :  IsUnit ((a : ZMod m) : ZMod n) := by
```

Comment:
```suggestion
lemma IsUnit.cast_of_dvd (hm : n ∣ m) (a : Units (ZMod m)) :  IsUnit ((a : ZMod m) : ZMod n) := by
```
Does dot notation work with this name?

Reply:
No, because there is no argument of type `IsUnit`.

Reply:
Oh, I misread `Units`, I am sorry!